vrrp/topology.yml

```diff
@@ -0,0 +1,84 @@
+#                                   +-------+
+#                                   |       |
+#                                   |   h1  |
+#                                   |       |
+#                                   +-------+
+#                                       | sw-host
+#                                       | 10.0.1.3/24
+#                                       |
+#                                       |
+#                                       |
+#                                       |
+#                                  +---------+
+#                                  |         |
+#             +--------------------|   sw    |-------------------+
+#             |                    |         |                   |
+#             |                    +---------+                   |
+#             |                                                  |
+#             |                                                  |
+#             |                                                  |
+#             |                                                  |
+#             |  eth-sw1                                         |  eth-sw1
+#             |  10.0.1.1/24                                     |  10.0.1.2/24
+#     +-------------+              +----------+             +------------+
+#     |             |              |          |             |            |
+#     |    RT-1     |              |  Virtual |             |     RT-2   |
+#     |   1.1.1.1   |==============|  Router  |=============|   2.2.2.2  |
+#     |             |              |  10.0.1.5|             |            |
+#     +-------------+              +----------+             +------------+
+#
+# KEY:
+# -----   physical link
+# =====   virtual link
+
+name: vrrp 
+
+topology:
+  defaults:
+    binds:
+      - ../scripts/start.sh:/start.sh 
+      - frr/frr-setup.sh:/frr-setup.sh      
+      - frr/daemons:/etc/frr/daemons
+    exec:
+      - "/start.sh"
+      - "/frr-setup.sh"
+
+  nodes:
+
+    sw1:
```

Comment:
Please use "br1" to be consistent with the other topologies. Similarly, please use "rt1" instead of "rt-1".

Reply:
Will do. 
Thanks.